nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -73,6 +74,22 @@
 
     protected static final String COMMENT_NAME = "comment";
 
+    /**
+     * Counts how many comments where made on a specific document.
+     *
+     * @since 11.1
+     */
+    protected static final String QUERY_GET_COMMENTS_COUNT_FOR_DOCUMENT_BY_COMMENT_ANCESTORS = //
+            "SELECT " + ECM_UUID + " FROM Comment WHERE " + COMMENT_ANCESTOR_IDS + "/* = '%s'";
+
+    /**
+     * Counts how many comments where made by a specific user on a specific document.
+     *
+     * @since 11.1
+     */
+    protected static final String QUERY_GET_COMMENTS_COUNT_FOR_DOCUMENT_BY_COMMENT_ANCESTORS_AND_AUTHOR = //
+            QUERY_GET_COMMENTS_COUNT_FOR_DOCUMENT_BY_COMMENT_ANCESTORS + " AND " + COMMENT_AUTHOR + " = '%s'";
+
```

Comment:
as these two constants are not used outer the class what if we just make them inner on method and reuse the method to get the first query on the second once

Reply:
in SQLInfo queries are saved in fields because they are computed at runtime.
We don't have this issue here and constants are clear and documented.
I'd rather keep them that way than making a method to recompute them everytime someone comments.

Reply:
yep if you remember we discussed with Florent on `SQLInfo` and he said that was not really needed and there is no read performance deal. My idea here is just to keep it simple the length of the name of these filed is big and they are protected they will be evaluated  anyway at runtime ... but this my point of view we can ask the team for more feedbacks :)

Reply:
I would keep it simple with the constants here...
WDYT @efge?

Reply:
I'd keep constants at the toplevel, even if they are relatively simple and used locally. Because these are NXQL queries and it's good to be able to see at the top of a class what kind of queries it does, and see them all together.
But the name could be shortened...

Reply:
QUERY_GET_COMMENTS_BY_COMMENT_ANCESTOR
QUERY_GET_COMMENTS_BY_COMMENT_ANCESTOR_AND_AUTHOR